packages/workbox-routing/src/_types.ts

```diff
@@ -47,10 +48,11 @@ import './_version.js';
  * @callback ~matchCallback
  * @param {Object} context
  * @param {URL} context.url The request's URL.
- * @param {Request} [context.request] The corresponding request,
- *     if available.
+ * @param {boolean} context.sameOrigin The result of comparing `url.origin`
+ * against the current origin.
+ * @param {Request} context.request The corresponding request.
  * @param {FetchEvent} [context.event] The corresponding event that triggered
- *     the request, if available.
+ * the request, if available.
```

Comment:
BTW, I indent these 4 spaces because that's what's recommended by the Google JS style (go/js then search for `@param`).

Reply:
I don't have a preference for which we do, just explaining why they're there (ESLint doesn't have an option to enforce).